source/RocketPlugin/GameModes/BoostPop.cpp

```diff
@@ -0,0 +1,149 @@
+// GameModes/BoostMod.cpp
+// A boost-type game mode for Rocket Plugin.
+//
+// Author:        Naoki95957
+// Version:       0.1.5 07/04/22
+// BMSDK version: 95
+
+//extra koodos to AL12 from BoostMod since I mostly moddified off that + some bois from the flock for the idea. 
+
+#include "BoostPop.h"
+
+
+/// <summary>Renders the available options for the game mode.</summary>
+void BoostPop::RenderOptions()
+{
+    // Team specific modifiers.
+    ImGui::TextUnformatted("Team modifiers:");
+    for (int teamIdx = 0; static_cast<size_t>(teamIdx) < boostModifierTeams.size(); teamIdx++) {
+        ImGui::PushID(teamIdx);
+        ImGui::Text(teamIdx ? "Orange Team" : "Blue Team", teamIdx);
```

Comment:
Keep this at `ImGui::Text("Team %d:", teamIdx);` because Orange Team and Blue Team are not always correct.